File: 4_catkin_ws/src/sensor_pkg/src/chao_node.cpp
#include <ros/ros.h>
#include <std_msgs/String.h>

int main(int argc, char *argv[])
{
    ros::init(argc, argv, "chao_node");

    //printf("Hello World!\n");
    //printf("你好 世界\n");

    ros::NodeHandle nh;
    ros::Publisher pub = nh.advertise<std_msgs::String>("Adiran_First_Topic",10);

    ros::Rate loop_rate(10);

    while(ros::ok())
    {
        printf("Hello World!\n");
        std_msgs::String msg;
        msg.data = "Adiran_First_msg";
        pub.publish(msg);
        loop_rate.sleep();
    }
    
    return 0;
}

File: 5_catkin_ws/src/sensor_pkg/scripts/chao_node.py
#!/usr/bin/env python3
#coding=utf-8

import rospy
from std_msgs.msg import String

if __name__ == "__main__":
    rospy.init_node("chao_node")
    rospy.logwarn("chao_node节点初始化成功！")

    pub = rospy.Publisher("Adiran_First_Topic",String,queue_size=10)

    rate = rospy.Rate(10)

    while not rospy.is_shutdown():
        rospy.loginfo("开始发送消息！")
        msg = String()
        msg.data = "Adiran_First_msg"
        pub.publish(msg)
        rate.sleep()
